system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,34 +320,89 @@
 }
 
 .topsite-form {
+  $form-width: 300px;
+  $form-spacing: 32px;
+
+  .form-input-container {
+    max-width: $form-width + 3 * $form-spacing + $rich-icon-size;
+    margin: 0 auto;
+    padding: $form-spacing $form-spacing 40px;
+
+    .top-site-outer {
+      padding: 0;
+      margin: 24px 0 0;
+      margin-inline-start: $form-spacing;
+      pointer-events: none;
+      cursor: default;
+    }
+
+    .section-title {
+      text-transform: none;
+      font-size: 16px;
+      line-height: 19px;
```

Comment:
This `line-height` doesn't seem to change anything with the `font-size: 16px`?

Reply:
It was supposed to share the style of https://github.com/mozilla/activity-stream/blob/a050adee0eebcc47c310ed9d9fb9973f1a07e1bd/system-addon/content-src/components/PreferencesPane/_PreferencesPane.scss#L68-L71